app/repos/people.js

```diff
@@ -35,6 +34,17 @@ const { personRelationship } = require('./relationships/person')
  * @property {AddressDao} address
  */
 
+/**
+ *
+ * @param {string} personReference
+ * @param transaction
+ * @returns {Promise<boolean>}
+ */
+const pkExists = async (personReference, transaction) => {
+  const count = await sequelize.models.person.count({ where: { person_reference: personReference } }, { transaction })
```

Comment:
Don't change this as very minor point, but in terms of effort in the DB, it is quicker to findOne (returning the first row it finds) than to do a count since the count may mean the DB has to read multiple rows. However, since we have a unique constraint on the personReference, the count could only ever return zero or 1, thereby the same effort as a findOne